common/list/_lists.scss

```diff
@@ -0,0 +1,163 @@
+/*
+ *
+ * Web Experience Toolkit (WET) / Boîte à outils de l'expérience Web (BOEW)
+ * wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ *
+ * @title: Lists
+ *
+ */
+
+/*
+ * List responsive style
+ *
+ * It has variant defined for some specific views
+ * - views/_screen-lg-min
+ * - views/_screen-xs-max
+ */
+.list-responsive {
+
+	> li {
+		float: left;
+		padding-right: 5px;
+		width: 50%;
+
+		&:nth-child(2n+2) {
+			clear:right;
+		}
+	}
+
+	&:before, &:after {
+		content: " ";
+		display: table;
+	}
+	&:after {
+		clear: both;
+	}
+}
+
+/* Double space list items
+	Might worth to publish it upstream here instead: https://github.com/wet-boew/wet-boew/blob/master/src/base/lists/_base.scss#L30
+ */
+div.lst-spcd-2>ul,
+div.lst-spcd-2>ol,
+.lst-spcd-2 {
+	> {
+		li {
+			margin-bottom: 20px;
+			margin-top: 20px;
+		}
+	}
+}
```

Comment:
This style is not retro-compatible. See the extra margin that is added on our existing list working example.

Before, a top margin estimated to: ~22px
![image](https://github.com/wet-boew/GCWeb/assets/1676030/fdb5cfb0-8cb9-45ed-9eac-5138d907a88e)

Now, with this update, that spaces is greater and estimated to: ~35px
![image](https://github.com/wet-boew/GCWeb/assets/1676030/b4a609f9-c1ef-4ed2-9d53-de11830baef2)

Reply:
Fixed. Thanks